app/models/state_file_archived_intake.rb

```diff
@@ -18,10 +22,88 @@
 #
 class StateFileArchivedIntake < ApplicationRecord
   has_one_attached :submission_pdf
+  devise :lockable, unlock_in: 60.minutes, unlock_strategy: :time
+  has_many :state_file_archived_intake_access_logs, class_name: 'StateFileArchivedIntakeAccessLog'
   has_many :state_file_archived_intake_requests, class_name: 'StateFileArchivedIntakeRequest'
 
+  before_create :populate_fake_addresses
   def full_address
     address_parts = [mailing_street, mailing_apartment, mailing_city, mailing_state, mailing_zip]
     address_parts.compact_blank.join(', ')
   end
+
+  def self.maximum_attempts
+    2
+  end
+
+  def increment_failed_attempts
+    super
+    lock_access! if attempts_exceeded? && !access_locked?
+  end
+
+  def fake_addresses
+    [fake_address_1, fake_address_2]
+  end
+
+  def address_challenge_set
+    (fake_addresses.push(full_address)).shuffle
+  end
+
+  private
+
+  # this is here because we don't want people to get new fake addresses if they refresh the page or return with a new session
```

Comment:
is this comment explaining why this method is here at all _or_ that it's `private`?